rotkehlchen/chain/manager.py

```diff
@@ -1078,7 +1078,11 @@ def query_ethereum_tokens(self) -> None:
         - EthSyncError if querying the token balances through a provided ethereum
         client and the chain is not synced
         """
-        evmtokens = EvmTokens(database=self.database, manager=self.ethereum)
+        evmtokens = EvmTokens(
+            database=self.database,
+            manager=self.ethereum,
+            makerdao_vaults_module=self.get_module('makerdao_vaults'),
```

Comment:
I did not check anything else but no this connetion is not gonna fly. Don't pass the makerdao vaults module to the EVMTokens class. That connection makes no sense. You are passing a specific ethereum module to a class designed as an abstract class for all EVMTokens.

Plus if it's touching files like this probaly this PR won't get merged soon.